### PR TITLE
Localization through lit-localize

### PR DESCRIPTION
**Pull request by hrobeers**
For a project at [Mahi](https://mahi.be/) based on openbridge-webcomponents components we are in need for Spanish translations.
To support this we included localization through lit-localize and added some initial Spanish translations for the components that we are currently using.

Please review and let's discuss how to best support localization.
This implementation is currently serving our needs.

### CODE REVIEW

File: packages/openbridge-webcomponents/src/generated/locales/es-419.ts
Comment (tibnor) [inline packages/openbridge-webcomponents/src/generated/locales/es-419.ts:1]: I think this file should be added to .gitignore. As it is autogenerated

**Review by tibnor — CHANGES_REQUESTED** [review]
Hi!

Thanks for the contribution! I think it would be very nice to have the library internationalized :-D I must read up on lit-localize, before I can merge it in. Do I need to do anything in the build and distribution?

File: packages/openbridge-webcomponents/src/generated/locales/es-419.ts
Comment (hrobeers) [reply packages/openbridge-webcomponents/src/generated/locales/es-419.ts:1]: Yes agreed, but that would also mean adding the `lit-localize build` step to the build process.

**Comment by hrobeers** [comment]
> Hi!
> 
> Thanks for the contribution! I think it would be very nice to have the library internationalized :-D I must read up on lit-localize, before I can merge it in. Do I need to do anything in the build and distribution?

I'm quite new to lit-localize as well, in a first stage it might be enough to just wrap all the to localize strings in `msg(string)`.
For our project this would be sufficient for now. I guess translations should come from final application and not from the library.

The idea of lit-localize is also that you can do all the wrapping of localized strings upfront and worry about the actual translations later.

Maybe we should keep it simple and just start wrapping strings?

**Review by tibnor — APPROVED** [review]

**Pull request merged**